src/worldstate/routine/Gate/include/alignWithGate.h

```diff
@@ -0,0 +1,22 @@
+//
+// Created by joel on 12/05/18.
+//
+
+#ifndef PROJECT_ALIGNWITHGATE_H
+#define PROJECT_ALIGNWITHGATE_H
+
+#include "State.h"
+#include <constants.h>
+#include <gate_detect/gateDetectMsg.h>
```

Comment:
@garethellis0 do we have a standard for messages? I feel like it should be the same as classes but we currently have `gateDetectMsg` and  `state_msg` so we should at least get them in line.

Reply:
ROS standard is CamelCase: http://wiki.ros.org/ROS/Patterns/Conventions#Messages

Reply:
Cool, let's get `state_msg` changed to `StateMsg` for this PR and we'll address the other one in the upcoming gate detection PR. @SoleSun

Reply:
Refactored!